pkg/pipeline/extract/conntrack/conntrack.go

```diff
@@ -181,7 +181,7 @@ func (ct *conntrackImpl) updateConnection(conn connection, flowLog config.Generi
 }
 
 func (ct *conntrackImpl) isLastFlowLogOfConnection(flowLog config.GenericMap) bool {
```

Comment:
I would remove `isLastFlowLogOfConnection` and `shouldSwapAB` since `containsTcpFlag` with the proper flag is better in terms of readability and these functions are only used in a single place

Reply:
Sure, no problem. If in the future we'll have these features for other protocols, we could add the abstraction layer.